src/netex_idf/offers.rs

```diff
@@ -883,4 +986,98 @@ mod tests {
             assert!(calendar.dates.contains(&Date::from_ymd(2019, 1, 2)));
         }
     }
+    mod stop_times {
+        use super::*;
+        use pretty_assertions::assert_eq;
+
+        #[test]
+        #[should_panic]
+        fn test_arrival_departure_times_invalid_xml() {
+            let tpt_xml = r#"<TimetabledPassingTime version="any"></TimetabledPassingTime>"#;
+            let tpt_el: Element = tpt_xml.parse().unwrap();
+            arrival_departure_times(&tpt_el).unwrap();
+        }
+
+        #[test]
+        fn test_arrival_departure_times_with_offset_0() {
+            let tpt_xml = r#"<TimetabledPassingTime version="any">
+                                        <ArrivalTime>01:30:00</ArrivalTime>
+                                        <DepartureTime>01:32:00</DepartureTime>
+                                        <DepartureDayOffset>0</DepartureDayOffset>
+                                    </TimetabledPassingTime>"#;
+            let tpt_el: Element = tpt_xml.parse().unwrap();
+            let times = arrival_departure_times(&tpt_el).unwrap();
+
+            let expected = (Time::new(0, 0, 5400), Time::new(0, 0, 5520));
```

Comment:
It'd be probably clearer to write it like this.

```suggestion
            let expected = (Time::new(1, 30, 0), Time::new(1, 32, 0));
```

Same comment for all the tests below.